.goreleaser.yml

```diff
@@ -28,6 +28,26 @@ brews:
     test: |
       system "#{bin}/dagger version"
 
+# https://goreleaser.com/customization/nix/
+nix:
+  - name: dagger
+    repository:
+      owner: dagger
+      name: nix
+    commit_author:
+      name: dagger-bot
+      email: noreply@dagger.io
+    path: pkgs/dagger/default.nix
+    url_template: "https://{{ .Env.ARTEFACTS_FQDN }}/dagger/releases/{{ .Version }}/{{ .ArtifactName }}"
+    homepage: "https://github.com/dagger/dagger"
+    description: "Dagger is a programmable deployment system"
```

Comment:
This differs from the GitHub repo description, but I copied it from the Homebrew tap section.

Reply:
That's OK for now, but we definitely want to be consistent.

What should this be @shykes @Neelajacques?

Reply:
Thank you @Neelajacques!

_Dagger is an integrated platform to orchestrate the delivery of applications_